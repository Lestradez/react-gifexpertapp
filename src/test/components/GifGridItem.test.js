import  GifExpertApp  from "../../GifExpertApp";
import React from 'react';
import { shallow } from "enzyme"; 
import '@testing-library/jest-dom';

describe ( 'Pruebas en <GifExpertApp />' , ()=>{
  let wrapper = shallow( <GifExpertApp/> );
   beforeEach(()=>{
    wrapper = shallow( <GifExpertApp/> );
  });

  test('Debe mostrar <GifGridItem /> correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  
  });

});