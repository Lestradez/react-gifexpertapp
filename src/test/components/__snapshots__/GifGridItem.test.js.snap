// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifExpertApp /> Debe mostrar <CounterApp /> correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;

exports[`Pruebas en <GifExpertApp /> Debe mostrar <GifGridItem /> correctamente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
